docs/src/tutorials/RK_order_conditions.md

```diff
@@ -0,0 +1,82 @@
+# Runge-Kutta Order conditions
+
+In this tutorial, we generate order conditions for a generic explicit Runge-Kutta method.
+
+First, we create symbolic coefficient arrays with the appropriate structure.
+
+
+```julia
+using BSeries, SymPy, Latexify
+
+s = 4  # Stages
+p = 4  # Desired order of accuracy
+
+A = Array{Sym,2}(undef,s,s)
+b = Array{Sym,1}(undef,s)
+c = Array{Sym,1}(undef,s);
+```
+
+
+```julia
+for i = 1:s
+    b[i] = symbols("b$i", real=true)
+    for j = 1:i-1
+        A[i,j] = symbols("a$i$j", real=true)
+    end
+    for j = i:s
+        A[i,j] = 0
+    end
+end
+
+for i = 1:s
+    c[i] = 0
+    for j = 1:i-1
+        c[i] += A[i,j]
+    end
+end
+```
+
+Next we generate the B-series for the RK method and the exact solution, and take their difference.
+
+
+```julia
+RK3 = bseries(A, b, c, p)
+exact = ExactSolution(RK3)
+error = RK3-exact
+```
```

Comment:
It is common to use lowercase for variables in Julia.

````suggestion
```@example bseries-RK-order-conditions

rk3 = bseries(A, b, c, p)
exact = ExactSolution(rk3)
error = rk3 - exact
```
````